src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -153,6 +152,10 @@ public static int Run(string[] args)
                 {
                     msbuildArgs.Add($"/verbosity:{verbosityOption.Value()}");
                 }
+                else
+                {
+                    msbuildArgs.Add("/verbosity:quiet");
```

Comment:
This will only add verbosity for the test verb right now.

Have you looked at MSBuildForwardingApp? It may be more appropriate for this.